projects/components/src/action-menu/action-menu.component.ts

```diff
@@ -118,52 +149,98 @@ export class ActionMenuComponent<R, T> {
      * List of only the actions that are marked as {@link ActionType.STATIC_FEATURED}
      */
     get staticFeaturedActions(): ActionItem<R, T>[] {
-        const staticActions = this.actions.filter(action => action.actionType === ActionType.STATIC_FEATURED);
-        return this.getAvailableActions(staticActions);
+        const staticFeaturedActions = this.actions.filter(action => action.actionType === ActionType.STATIC_FEATURED);
+        return this.getAvailableActions(staticFeaturedActions);
     }
 
-    private getFlattenedActionList(actions: ActionItem<R, T>[], actionType: ActionType): ActionItem<R, T>[] {
-        let featuredActions: ActionItem<R, T>[] = [];
-        actions.forEach(action => {
+    /**
+     * List containing all the static actions. It has static featured actions in the beginning of the list followed by
+     * non-featured static actions as children of grouped action called 'vcd.cc.action.menu.nonFeatured.actions'
+     */
+    get staticDropdownActions(): ActionItem<R, T>[] | object {
+        return this.staticFeaturedActions.concat([{ textKey: 'Non featured', children: this.staticActions }]);
```

Comment:
Is 'Non featured' being displayed? if so, it should be translated (either input from the user, or some key that we know about and give a default translation for)